modules/signing_worker/files/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e git+https://github.com/escapewindow/scriptworker-scripts#egg=scriptworker_client&subdirectory=scriptworker_client
+-e git+https://github.com/escapewindow/scriptworker-scripts#egg=iscript&subdirectory=iscript
```

Comment:
I added the vendored/mozbuild dir as a requirement.
I'm guessing we may want to copy all of these non-pypi modules up to s3 once they're in a released state.